FA22.P05.Web/ClientApp/src/pages/Listings/listings.tsx

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
is this function being used, is so remove all the extra lines and if not delete it